gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallableFactory.java

```diff
@@ -85,7 +85,9 @@ public static <RequestT, ResponseT> UnaryCallable<RequestT, ResponseT> createBas
         GrpcRawCallableFactory.createUnaryCallable(
             grpcCallSettings, callSettings.getRetryableCodes());
 
-    callable = Callables.retrying(callable, callSettings, clientContext);
+    callable =
```

Comment:
add check that if requestMutator exists, then call this method with the additional 4th parameter. otherwise, use the original method (with 3 parameters).